zhaquirks/sonoff/swv.py

```diff
@@ -56,5 +62,13 @@ class AttributeDefs(BaseAttributeDefs):
         translation_key="water_supply",
         fallback_name="Water supply",
     )
+    .switch(
+        CustomSonoffCluster.AttributeDefs.auto_close_when_water_shortage.name,
+        CustomSonoffCluster.cluster_id,
+        off_value=0,
+        on_value=30,
```

Comment:
What does value `30` mean for the device? Like, what do other values do for example?